Content/Tiles/ArcaneObeliskTile.cs

```diff
@@ -0,0 +1,88 @@
+using PathOfTerraria.Common.Waypoints.UI;
+using PathOfTerraria.Content.Items.Placeable;
+using PathOfTerraria.Core.UI;
+using ReLogic.Content;
+using Terraria.DataStructures;
+using Terraria.GameContent;
+using Terraria.GameContent.ObjectInteractions;
+using Terraria.ID;
+using Terraria.ObjectData;
+
+namespace PathOfTerraria.Content.Tiles;
+
+public class ArcaneObeliskTile : ModTile
+{
+	public override void SetStaticDefaults()
+	{
+		base.SetStaticDefaults();
+
+		Main.tileLavaDeath[Type] = false;
+		Main.tileWaterDeath[Type] = false;
+		Main.tileBlockLight[Type] = true;
+		Main.tileFrameImportant[Type] = true;
+
+		TileID.Sets.HasOutlines[Type] = true;
+
+		TileObjectData.newTile.CopyFrom(TileObjectData.Style3x4);
```

Comment:
More of an opinion question; why newlines between some TileObjectData members? I usually have a continuous block for TileObjectData.newTile stuff, such that CopyFrom to addTile have no whitelines. I personally think that's much easier to read, but if you have some format in your head that makes more sense to you that's fair.

Reply:
Usually I just try to decouple big chunks of statements by associating them with what they do, so In the snippet you saw for example I did it from visuals, coordinates and placement.